contracts/src/Rollup.sol

```diff
@@ -64,6 +64,34 @@ abstract contract RollupBase is
         __Ownable_init();
         __UUPSUpgradeable_init();
     }
+
+    /// @inheritdoc IRollup
+    function setConfirmationPeriod(uint256 newPeriod) public override onlyOwner {
+        confirmationPeriod = newPeriod;
+        emit ConfigurationChanged();
+    }
+
+    /// @inheritdoc IRollup
+    function setChallengePeriod(uint256 newPeriod) public override onlyOwner {
```

Comment:
Change the function from public to external perhaps? That is the function signature in interface as well as the gas-efficient way to move forward assuming this function is not called inside the contracts.